_posts/2020-12-31-test.md

```diff
@@ -4,6 +4,11 @@ title:  "Happy Bobo"
 description: "Lorem markdownum e Daphnes convocat perque medios, pro nigra totiens. Maledicere tenebas. Manus subit moderantum sunt iamdudum qui exsilit dictis forte numen vulgus facilem. Acta honore eandem"
 image: bobo.jpg
 date: 20/10/2020
+authors:
+    - name: "Bobo Mono"
+      email: "bobo@riot-os.org"
+      githubid: "riot-ci"
```

Comment:
Should we maybe also add a link to the forum?

Reply:
not in favor of this as i don't see this such links as common.  you can refer in the text if needed.

it is rather more likley that you link from the forum to thie blog article.

Reply:
But what if the reader prefers to contact the author via discourse DM? Or the author prefers a discourse DM vs publishing their email address?

Reply:
At any rate: all of these should be kept optional of course, I just want to give authors the ability to be contacted the way they prefer in a manner that integrates nicely into our blog.

Reply:
We also could add more options, like they do here: https://qfimg.github.io/author-box-jekyll/

Reply:
Extended to support the following:
- web
- twitter
- stack overflow
- reddit
- linkedin
- mastodon
- facebook

Reply:
Still no discourse :disappointed:, but okay.